templates/blog/post_list.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  <title>Blog Posts</title>
+{% endblock %}
+
+{% block content %}
+  <h2>Posts</h2>
+  <div>
+    <ul>
+      {% for post in post_list %}
+        <li>
+          <h3><a href="{% url 'blog:post-detail' pk=post.id %}">{{ post.title }}</a></h3>
+          <div> Author: {{ post.owner }}</div>
+          <div>{{ post.created_time }}</div>
+          <div>{{ post.content }}</div>
+          <div>Commentaries: {{ post.commentary.count }}</div>
```

Comment:
The text 'Commentaries' might be better replaced with 'Comments'. 'Commentaries' is usually used in the context of a detailed explanation or interpretation, while 'Comments' refers to brief notes or observations.